docker/package/model.py

```diff
@@ -530,6 +530,69 @@ def __init__(
                     prestart_script="tezos-baking-prestart",
                 )
             )
+        custom_requires = []
+        for network in target_networks:
+            for proto in network_protos[network]:
+                custom_requires.append(f"tezos-baker-{proto.lower()}@custom@%i.service")
+                if proto not in self.noendorser_protos:
+                    custom_requires.append(
+                        f"tezos-endorser-{proto.lower()}@custom@%i.service"
+                    )
+        self.systemd_units.append(
+            SystemdUnit(
+                service_file=ServiceFile(
+                    Unit(
+                        after=["network.target"],
+                        requires=["tezos-node-custom@%i"] + custom_requires,
+                        description=f"Tezos baking instance for custom network",
+                    ),
+                    Service(
+                        exec_start="/usr/bin/tezos-baking-start",
+                        user="tezos",
+                        state_directory="tezos",
+                        environment_file=f"/etc/default/tezos-baking-custom@%i",
+                        exec_start_pre=[
+                            "+/usr/bin/setfacl -m u:tezos:rwx /run/systemd/ask-password",
+                            "/usr/bin/tezos-baking-prestart",
+                        ],
+                        exec_stop_post=[
+                            "+/usr/bin/setfacl -x u:tezos /run/systemd/ask-password"
+                        ],
+                        remain_after_exit=True,
+                        type_="oneshot",
+                        keyring_mode="shared",
+                    ),
+                    Install(wanted_by=["multi-user.target"]),
+                ),
+                instances=[],
+                suffix="custom",
+                config_file="tezos-baking-custom.conf",
+                startup_script="tezos-baking-start",
+                prestart_script="tezos-baking-prestart",
+            )
+        )
+        self.systemd_units.append(
+            SystemdUnit(
+                service_file=ServiceFile(
+                    Unit(
+                        after=[],
+                        description=f"One-shot service to reset custom baking service",
+                    ),
+                    Service(
+                        exec_start="/usr/bin/tezos-baking-custom-reset %i",
+                        user="root",
+                        state_directory="tezos",
+                        environment_file=f"/etc/default/tezos-baking-custom@%i",
```

Comment:
Does it make sense that its `environment_file` is the same as for the custom baking service itself?

Reply:
Hmm, seems like I initially wanted to use it in order to determine `NODE_DATA_DIR` from it, but at the moment `tezos-baking-custom-reset` is determining this env variable from `systemctl show`.
Actually, it's questionable what is better. On one hand `environment_file` here is indeed redundant, on the other hand, it's a more reliable way to determine env variable from this file rather than `grep`'ng `systemctl show` output.

What do you think?

Reply:
Reliable is better 🙈 One redundant option doesn't matter that much in terms of performance nor readability, I think, so I'd use the file.

Reply:
Oh, wait, I was wrong `NODE_DATA_DIR` isn't defined in this `environment_file`:(
So this option is indeed redundant here